src/graph_notebook/notebooks/03-Sample-Applications/README.ipynb

```diff
@@ -0,0 +1,59 @@
+    "\n",
+    "Knowledge graphs consolidate and integrate an organization’s information assets and make them more readily available to all members of the organization. There are many applications and use cases that are enabled by knowledge graphs. Information from disparate data sources can be linked and made accessible to answer questions you may not even have thought of yet. Information and entities can be extracted not only from structured sources (e.g., relational databases) but also from semi-structured sources (e.g., media, metadata, and spreadsheets) and unstructured sources (e.g., text documents, email, and news articles).\n",
+    "\n",
+    "The examples in this notebook show how we can use our blog knowledge graph to demonstrate how we can use the connected nature of our knowledge graph to provide contextually relevant answers to search questions.\n",
+    "\n",
+    "## [Identity Graph](./03-Identity-Graphs/01-Building-an-Identity-Graph-Application.ipynb)\n",
+    "\n",
+    "An identity graph provides a single unified view of customers and prospects by linking multiple identifiers such as cookies, device identifiers, IP addresses, email IDs, and internal enterprise IDs to a known person or anonymous profile using privacy-compliant methods. Typically, identity graphs are part of a larger identity resolution architecture. Identity resolution is the process of matching human identity across a set of devices used by the same person or a household of persons for the purposes of building a representative identity, or known attributes, for targeted advertising.\n",
+    "\n",
+    "The examples in this notebook walk you through a sample solution for identity graph and how it can be used within a larger identity resolution architecture using an open dataset and the use of a graph database, Amazon Neptune. In this notebook, we also show a number of data visualizations that allow one to better understand the structure of an identity graph and the aspects of an identity resolution dataset and use case. We also showcase some additional use cases that can be exposed using this particular dataset."
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.11"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
For some reason, this doesn't render properly for me when I go to see it here: https://github.com/aws/graph-notebook/tree/austinkline/notebookcleanup/src/graph_notebook/notebooks/03-Sample-Applications

Can we change it to a README.md potentially?

Reply:
While github might not render this properly, changing it to README.md will not render in jupyter which I would say is higher on the priority (see screenshots)

README.md
![image](https://user-images.githubusercontent.com/8711160/109742546-17cad380-7b84-11eb-995b-17cef4af5376.png)

README.ipynb
![image](https://user-images.githubusercontent.com/8711160/109742583-2618ef80-7b84-11eb-851a-afaa355707fb.png)